### PR TITLE
Document l10n-changesets.json in sign-off document

### LINKED ISSUE #202: Sign-off documentation: l10n-changesets, release update, Fennec
While reviewing a patch with nits, I recognized that the documentation doesn't reflect reality as we have it.

For one, the role of Sign-offs on Firefox 4 Android should be clarified. I looked at the sign-off page, and got very confused. Also, on Android, we ship default on beta, as part of geckoview.

The role of `l10n-changesets.json` isn't described, which makes the language about beta and nightly hard to grasp.

The procedure to update sign-offs on release is missing.  More so, the documentation suggests that it doesn't exist.

### PR DESCRIPTION
**Pull request by flodolo**
Fixes #202

### CODE REVIEW

**Comment by flodolo** [comment]
As explained in the issue, I don't think we should drop mentions of Fennec for now, although it's only a couple of links and some text.

We should also not document what is happening for Fennec in more details (the whole mapping to `68.(beta-X).0`), because that's a temporary solution, and we're going to drop it in a couple of months.

File: src/products/review/signoffs.md [hunk 1]
```diff
@@ -19,16 +21,22 @@ Sign-offs on Beta are not possible through the entire cycle: sign-offs deadline
...
 * Since sign-offs are tied to a version number, when Nightly moves to Beta, the new Beta version will already have updated sign-offs and won’t fall back to outdated changesets. As a consequence, the first beta of the cycle will ships with better l10n coverage. In the example above, you start taking sign-offs for 77 at the end of the Nightly cycle. When 77 moves to Beta, it will already have update content compared to the last 76 sign-offs.
 
+### Updating release
+
+When the code moves from `mozilla-beta` to `mozilla-release`, `l10n-changesets.json` is frozen, as `l10n-bumper` is not configured to run against the release branch, and Elmo doesn’t allow to sign off changesets on the release channel.
+
+In case of severe issues affecting one or more locales, it’s still possible to manually update the shipping changesets. A patch needs to be provided for `l10n-changesets.json` in `mozilla-release` branch and approved for uplift by Release Drivers. Note that a dot release is needed in order to ship the updated version to users.
```
Comment (flodolo) [inline src/products/review/signoffs.md]: This seems vague, but I tried to find examples, but I couldn't find any. All the cases I've found were still using Ship-It, and it was a matter of asking people to update the shipping changeset.

Comment (flodolo) [reply src/products/review/signoffs.md]: And I've figured out how to find one right after writing this comment 🤦

**Review by Pike — APPROVED** [review]
Thanks, this looks much better.

**Comment by flodolo** [comment]
Thanks, hopefully I'm done monopolizing your documentation time :)

**Pull request merged**